administration/src/bp-modules/stores/StoresController.tsx

```diff
@@ -26,6 +26,15 @@ const StoresController = (): ReactElement => {
       />
     )
   }
+  if (!storeManagement.enabled) {
+    return (
+      <NonIdealState
+        icon='cross'
+        title='Fehlende Berechtigung'
+        description='Die Verwaltung der Akzeptanzpartner ist nicht aktiviert.'
+      />
+    )
+  }
```

Comment:
In theory not necessary to add a separate if clause:
```suggestion
if (role !== Role.ProjectStoreManager || !storeManagement.enabled) {
    return (
      <NonIdealState
        icon='cross'
        title='Fehlende Berechtigung'
        description={!storeMangment.enabled ? 'Die Verwaltung der Akzeptanzpartner ist nicht aktiviert.' : Sie sind nicht berechtigt, Akzeptanzpartner zu verwalten.}
      />
    )
  }
```